tests/chainer_tests/test_reporter.py

```diff
@@ -216,5 +216,20 @@ def test_float(self):
         testing.assert_allclose(mean, 2.)
         testing.assert_allclose(std, numpy.sqrt(2. / 3.))
 
+    def test_serialize(self):
+        self.summary.add(1.)
+        self.summary.add(2.)
+
+        resumed_summary = chainer.reporter.Summary()
+        testing.save_and_load_npz(self.summary, resumed_summary)
+        resumed_summary.add(3.)
+
+        mean = resumed_summary.compute_mean()
+        testing.assert_allclose(mean, 2.)
+
+        mean, std = resumed_summary.make_statistics()
+        testing.assert_allclose(mean, 2.)
+        testing.assert_allclose(std, numpy.sqrt(2. / 3.))
+
 
```

Comment:
Could you please add tests for `DictSummary` class?
(It is acceptable to only write test for the serialization part you added.)